packaging/test-release-packages.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 set -eux
 # Verify package install for a latest release version
+docker run --rm -it ubuntu:22.04 sh -c "apt-get update && apt-get install -y sudo gpg curl;curl https://raw.githubusercontent.com/fluent/fluent-bit/master/install.sh | sh"
```

Comment:
Did you test your package with a container install? This will not work currently as the repos need adding - that's a separate task on the original issue we need to do.

Reply:
Hey @patrick-stephens need help with this step. How do I run the test with apt? is there any documentation available on this?

Reply:
https://unix.stackexchange.com/a/159114

Basically this script tries to install from the official repo which doesn't exist so you're going to have to mount it in and do a local install. Or create a test repo and update the script to use it.

Reply:
See #5444 for some improvements you may want to pick up as well.